kubernetes/k8s-listener.py

```diff
@@ -317,7 +353,11 @@ def get_segment_id_by_name(name, segments):
 }
 
 
+# Processing kubernetes events coming from api
 def process(s):
+
+    # Kube api listener is a GET request which will timeout eventually
+    # producing empty request.
```

Comment:
Well, if it's timing out then it's more than just an empty line, it's a broken connection. So, some additional code is needed to deal with that (task #263).